tools/webdriver/webdriver/bidi/modules/input.py

```diff
@@ -1,22 +1,13 @@
 from collections import defaultdict
 
-from typing import (Any,
-                    ClassVar,
-                    List,
-                    Mapping,
-                    MutableMapping,
-                    Optional,
-                    Sequence,
-                    Set,
-                    Type,
-                    TypeVar,
-                    Union)
+from typing import (Any, ClassVar, List, Mapping, MutableMapping, Optional,
+                    Sequence, Set, Type, TypeVar, Union)
```

Comment:
Why this change? It's actually the opposite what you did in line 129. And I think this change is no correct.